### PR TITLE
Version bump 

### PR DESCRIPTION
**Pull request by samgreen**

### CODE REVIEW

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -278,7 +278,7 @@ PODS:
   - Yoga (1.14.0)
 
 DEPENDENCIES:
-  - berbix-react-native (from `../.yalc/berbix-react-native`)
+  - berbix-react-native (from `../node_modules/berbix-react-native`)
```
Comment (samgreen) [inline example/ios/Podfile.lock:281]: This seems like I might have done something wrong.

Comment (benbeltzer) [reply example/ios/Podfile.lock:281]: hmmmm. I'm honestly not quite sure what to make of that. did you do an `npm i` at some point?

Comment (samgreen) [reply example/ios/Podfile.lock:281]: Highly possible given recent investigations.

Comment (samgreen) [reply example/ios/Podfile.lock:281]: A fresh clone and `yarn bootstrap` gives the same results.

**Review by benbeltzer — APPROVED** [review]

**Pull request merged**